build/configs/artik053/README.md

```diff
@@ -17,6 +17,70 @@ Aimed especially at power-sensitive devices needing Wi-Fi®, the ARTIK 053 Modul
 
 Here is a [memory map](scripts/README.md).
 
+## ARTIK053 Starter Kit PinOut
```

Comment:
This is an one of information. So, you need to use "###". But if you think this is enough importatnt to keep it as a big title, please add this in contents.